prepare.ts

```diff
@@ -149,6 +149,12 @@ export default function prepare(
         "X-Vault-Token": VAULT_TOKEN,
       },
     });
+    if (response.statusCode == 403) {
+      console.error(
+        "vault responded with 403 access denied when i tried to rotate, giving up"
+      );
+      return;
+    }
     if (sync) {
       onLease(vaultPath, parseLeaseResponse(response));
```

Comment:
Does `onLease` re-install the timer on the happy path? Once you print the error, we should make sure that the cred rolling callback stops running, either by not reinstalling it, or by stopping it if it's a periodic timer.

Reply:
@emk do you mean stopping all future invocations of onLease?